stable/contour/templates/clusterrole.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.rbac.create -}}
+
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "contour.fullname" . }}
+  labels:
+    app: {{ template "contour.fullname" . }}
+    chart: {{ template "contour.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
```

Comment:
I apologize for "moving the goal posts on you" but the format for the labels has changed because of the way upstream Kubernetes not recommends using them. See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Reply:
Fixed the labels according to k8s recommendations